VERSION

```diff
@@ -1 +1 @@
-0.3.0
+0.3.1
```

Comment:
mind reverting this bump until we are ready for a release? the jeweler gem will take care of this for us when we release- which should actually be 0.4.0 with this addition.